migration/mule/import-contacts-asynchronously/SalesforceImportContactsAsynchronously/src/main/synapse-config/api/SalesforceImportContactAsynchronously.xml

```diff
@@ -0,0 +1,87 @@
+                <property expression="json-eval($.contentType)" name="contentType" scope="default" type="STRING"/>
+                <property expression="json-eval($.contentType)" name="contentType" scope="default" type="STRING"/>
+                <property name="conf:/resource/importContactStatus" scope="registry" type="STRING" value="In Progress"/>
+                <property expression="fn:substring-after(get-property('MessageID'), 'urn:uuid:')" name="MessageID" scope="default" type="STRING"/>
+            </propertyGroup>
+            <clone>
+                <target>
+                    <sequence>
+                        <salesforcerest.init>
+                            <accessToken>{$ctx:accessToken}</accessToken>
+                            <apiVersion>{$ctx:apiVersion}</apiVersion>
+                            <hostName>{$ctx:hostName}</hostName>
+                            <refreshToken>{$ctx:refreshToken}</refreshToken>
+                            <apiUrl>{$ctx:apiUrl}</apiUrl>
+                            <registryPath>{$ctx:registryPath}</registryPath>
+                            <intervalTime>{$ctx:intervalTime}</intervalTime>
+                            <blocking>{$ctx:blocking}</blocking>
+                        </salesforcerest.init>
+                        <log level="full" separator=","/>
+                        <fileconnector.read>
+                            <source>{$ctx:source}</source>
+                            <contentType>{$ctx:contentType}</contentType>
+                            <filePattern>{$ctx:filePattern}</filePattern>
+                        </fileconnector.read>
+                        <log level="full" separator=","/>
+                        <datamapper config="gov:datamapper/MapCSV.dmc" inputSchema="gov:datamapper/MapCSV_inputSchema.json" inputType="CSV" outputSchema="gov:datamapper/MapCSV_outputSchema.json" outputType="JSON" xsltStyleSheet="gov:datamapper/MapCSV_xsltStyleSheet.xml"/>
+                        <log level="full" separator=","/>
+                        <property expression="json-eval($.fieldAndValue)" name="fieldAndValue" scope="default" type="STRING"/>
+                        <salesforcerest.createMultipleRecords>
+                            <sObjectName>{$ctx:sObjectName}</sObjectName>
+                            <fieldAndValue>{$ctx:fieldAndValue}</fieldAndValue>
+                        </salesforcerest.createMultipleRecords>
+                        <script language="js"><![CDATA[java.lang.Thread.sleep(20000);]]></script>
+                        <property name="conf:/resource/importContactStatus" scope="registry" type="STRING" value="Done"/>
+                    </sequence>
+                </target>
+                <target>
+                    <sequence>
+                        <payloadFactory media-type="json">
+                            <format>
+						      	"Job ID" : "$1"
```

Comment:
Formatting issue. Please fix.

Reply:
Ideally, we need to have below also in the response payload. 
"Synchronization triggered at" : "$2"

$2 == now()